core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -957,7 +964,18 @@ def _train_and_save(self, X, y, model: AbstractModel, X_val=None, y_val=None, st
                     time_left_total = time_limit
                 fit_log_message += f' Training model for up to {round(time_limit, 2)}s of the {round(time_left_total, 2)}s of remaining time.'
             logger.log(20, fit_log_message)
-            model = self._train_single(X, y, model, X_val, y_val, **model_fit_kwargs)
+
+            # If model is not bagged model and not stacked then pseudolabeled data needs to be incorporated at this level
+            # Bagged model does validation on the fit level where as single models do it separately. Hence this if statement
+            # is required
+            if not isinstance(model, BaggedEnsembleModel) and X_pseudo is not None and y_pseudo is not None and X_pseudo.columns.equals(X.columns):
```

Comment:
this is for the case that X is the stacked predictions, because for single instance models sometimes it's trained using stacked predictions. So X needs to checked again.

Reply:
@Innixma is this issue resolved?

Reply:
@jwmueller No, but it is beyond the scope of this PR.